### DISCUSSION TITLE
done with lesson 1

### DISCUSSION THREAD

**Discussion by Lamma-maihadisi** [Cyfrin/foundry-full-course-cu#2652] [General]
hi , i'm super excited for taking this great course. thank you

**Comment by BowtiedHarpyEagle** [comment]
Great job. Keep on crushing!

**Reply by Lamma-maihadisi** [reply]
ok sir, thanks for caring.

**Comment by EngrPips** [comment]
Welcome aboard @Lamma-maihadisi
